roiextractors/segmentationextractor.py

```diff
@@ -3,6 +3,9 @@
 import numpy as np
 from .extraction_tools import ArrayType
 from .extraction_tools import _pixel_mask_extractor
+from copy import deepcopy
+import warnings
```

Comment:
these imports appear to be unused
```suggestion
```

Reply:
yes, they are unused in this PR. Fixed https://github.com/catalystneuro/roiextractors/pull/44/commits/4562f963d1d9d9aabd073c2ea2acba9d61eb594b